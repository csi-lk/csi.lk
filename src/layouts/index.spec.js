import React from 'react'

import Index from './index'

describe('Layout: Index', () => {
  it('renders', () => {
    const wrapper = shallow(<Index>{() => jest.fn()}</Index>)
    expect(wrapper).toMatchSnapshot()
  })
})
