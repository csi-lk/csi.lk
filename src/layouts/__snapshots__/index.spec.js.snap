// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout: Index renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TemplateWrapper>
    [Function]
  </TemplateWrapper>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
        >
          <meta
            content="Callum Silcock, contact@csi.lk"
            name="author"
          />
          <meta
            content="contact@csi.lk"
            name="reply-to"
          />
          <meta
            content="https://www.csi.lk"
            name="url"
          />
          <meta
            content="Worldwide"
            name="coverage"
          />
          <meta
            content="Global"
            name="distribution"
          />
          <meta
            content="General"
            name="rating"
          />
          <meta
            content="7 days"
            name="revisit-after"
          />
          <meta
            content="0"
            httpEquiv="Expires"
          />
          <meta
            content="no-cache"
            httpEquiv="Pragma"
          />
          <meta
            content="no-cache"
            httpEquiv="Cache-Control"
          />
          <meta
            content="en"
            httpEquiv="Content-Language"
          />
          <meta
            content="text/html"
            name="og:type"
          />
          <meta
            content="Callum Silcock | Senior UI Developer"
            name="og:site_name"
          />
          <meta
            content="contact@csi.lk"
            name="og:email"
          />
          <meta
            content="+61412454084"
            name="og:phone_number"
          />
        </HelmetWrapper>,
        <div>
          [MockFunction]
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <meta
              content="Callum Silcock, contact@csi.lk"
              name="author"
            />,
            <meta
              content="contact@csi.lk"
              name="reply-to"
            />,
            <meta
              content="https://www.csi.lk"
              name="url"
            />,
            <meta
              content="Worldwide"
              name="coverage"
            />,
            <meta
              content="Global"
              name="distribution"
            />,
            <meta
              content="General"
              name="rating"
            />,
            <meta
              content="7 days"
              name="revisit-after"
            />,
            <meta
              content="0"
              httpEquiv="Expires"
            />,
            <meta
              content="no-cache"
              httpEquiv="Pragma"
            />,
            <meta
              content="no-cache"
              httpEquiv="Cache-Control"
            />,
            <meta
              content="en"
              httpEquiv="Content-Language"
            />,
            <meta
              content="text/html"
              name="og:type"
            />,
            <meta
              content="Callum Silcock | Senior UI Developer"
              name="og:site_name"
            />,
            <meta
              content="contact@csi.lk"
              name="og:email"
            />,
            <meta
              content="+61412454084"
              name="og:phone_number"
            />,
          ],
          "defer": true,
          "encodeSpecialCharacters": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "Callum Silcock, contact@csi.lk",
              "name": "author",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "contact@csi.lk",
              "name": "reply-to",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "https://www.csi.lk",
              "name": "url",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "Worldwide",
              "name": "coverage",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "Global",
              "name": "distribution",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "General",
              "name": "rating",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "7 days",
              "name": "revisit-after",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "0",
              "httpEquiv": "Expires",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "no-cache",
              "httpEquiv": "Pragma",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "no-cache",
              "httpEquiv": "Cache-Control",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "en",
              "httpEquiv": "Content-Language",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "text/html",
              "name": "og:type",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "Callum Silcock | Senior UI Developer",
              "name": "og:site_name",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "contact@csi.lk",
              "name": "og:email",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "+61412454084",
              "name": "og:phone_number",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": [MockFunction],
        },
        "ref": null,
        "rendered": [MockFunction],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
          >
            <meta
              content="Callum Silcock, contact@csi.lk"
              name="author"
            />
            <meta
              content="contact@csi.lk"
              name="reply-to"
            />
            <meta
              content="https://www.csi.lk"
              name="url"
            />
            <meta
              content="Worldwide"
              name="coverage"
            />
            <meta
              content="Global"
              name="distribution"
            />
            <meta
              content="General"
              name="rating"
            />
            <meta
              content="7 days"
              name="revisit-after"
            />
            <meta
              content="0"
              httpEquiv="Expires"
            />
            <meta
              content="no-cache"
              httpEquiv="Pragma"
            />
            <meta
              content="no-cache"
              httpEquiv="Cache-Control"
            />
            <meta
              content="en"
              httpEquiv="Content-Language"
            />
            <meta
              content="text/html"
              name="og:type"
            />
            <meta
              content="Callum Silcock | Senior UI Developer"
              name="og:site_name"
            />
            <meta
              content="contact@csi.lk"
              name="og:email"
            />
            <meta
              content="+61412454084"
              name="og:phone_number"
            />
          </HelmetWrapper>,
          <div>
            [MockFunction]
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <meta
                content="Callum Silcock, contact@csi.lk"
                name="author"
              />,
              <meta
                content="contact@csi.lk"
                name="reply-to"
              />,
              <meta
                content="https://www.csi.lk"
                name="url"
              />,
              <meta
                content="Worldwide"
                name="coverage"
              />,
              <meta
                content="Global"
                name="distribution"
              />,
              <meta
                content="General"
                name="rating"
              />,
              <meta
                content="7 days"
                name="revisit-after"
              />,
              <meta
                content="0"
                httpEquiv="Expires"
              />,
              <meta
                content="no-cache"
                httpEquiv="Pragma"
              />,
              <meta
                content="no-cache"
                httpEquiv="Cache-Control"
              />,
              <meta
                content="en"
                httpEquiv="Content-Language"
              />,
              <meta
                content="text/html"
                name="og:type"
              />,
              <meta
                content="Callum Silcock | Senior UI Developer"
                name="og:site_name"
              />,
              <meta
                content="contact@csi.lk"
                name="og:email"
              />,
              <meta
                content="+61412454084"
                name="og:phone_number"
              />,
            ],
            "defer": true,
            "encodeSpecialCharacters": true,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "Callum Silcock, contact@csi.lk",
                "name": "author",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "contact@csi.lk",
                "name": "reply-to",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "https://www.csi.lk",
                "name": "url",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "Worldwide",
                "name": "coverage",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "Global",
                "name": "distribution",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "General",
                "name": "rating",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "7 days",
                "name": "revisit-after",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "0",
                "httpEquiv": "Expires",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "no-cache",
                "httpEquiv": "Pragma",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "no-cache",
                "httpEquiv": "Cache-Control",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "en",
                "httpEquiv": "Content-Language",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "text/html",
                "name": "og:type",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "Callum Silcock | Senior UI Developer",
                "name": "og:site_name",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "contact@csi.lk",
                "name": "og:email",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "+61412454084",
                "name": "og:phone_number",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": [MockFunction],
          },
          "ref": null,
          "rendered": [MockFunction],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
